modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/RichText/RichText.es.js

```diff
@@ -181,14 +196,17 @@ const RichText = ({
 			...newEditingLocale,
 			icon: normalizeLocaleId(newEditingLocale.localeId),
 		});
-		setCurrentInternalValue(
-			getEditingValue({
-				defaultLocale,
-				editingLocale: newEditingLocale,
-				fieldName,
-				value: newCurrentValue,
-			})
-		);
+
+		if (!Liferay.FeatureFlags['LPD-11235']) {
+			setCurrentInternalValue(
+				getEditingValue({
+					defaultLocale,
+					editingLocale: newEditingLocale,
+					fieldName,
+					value: newCurrentValue,
+				})
+			);
+		}
```

Comment:
Hi @thektan, the new [useEffect](https://github.com/liferay-bpm/liferay-portal/pull/5431/commits/e56aaac15796085af12f5bff9bfb299abd3aadff#diff-133dbaa1aab879d3c947647ceab0d40cf38f8e64bcfe327535c39a935c61be9cR153) doesn't cover this case if we remove the `if (Liferay.FeatureFlags['LPD-11235'])` condition?